### ISSUE TITLE
Which dotnet version?

### ISSUE THREAD

**Issue by tgestabrook** [LANDIS-II-Foundation/Core-Model-v8-LINUX#2]
Which version of the .NET sdk is required for running LANDIS-v8 on Linux? My understanding is that v8 uses [.NET version 8](https://www.landis-ii.org/install#:~:text=We%20updated%20.NET%20to%20use%20the%20latest%20version%208.), but the screenshot provided in the ReadMe looks like it's indicating version 2.1.403.

**Comment by achubaty** [comment]
Yes, .NET 8.0 is correct - the documentation simply hasn't been updated in all places to reflect that.

https://github.com/LANDIS-II-Foundation/Core-Model-v8-LINUX/blob/8725da5cd7bfdf3ae28bbd59b1353ec442783d68/Tool-Console/src/Console.csproj#L4
